### DISCUSSION TITLE
🐞 docs: h1 and h2 have the same size

### DISCUSSION THREAD

**Discussion by levlaz** [dagger/dagger#12271] [Help]
### What is the issue?

While reviewing a PR I noticed that H1 and H2 appear to have the same font-size, looking in chrome dev tools I see some conflicting rules for h2, it looks like its being set to 2rem instead of using the variables. 

```
h1  font-size: var(--ifm-h1-font-size);
```

```
.markdown>h2 {
    --ifm-h2-font-size: 2rem;
    margin-top: calc(var(--ifm-h2-vertical-rhythm-top)*var(--ifm-leading));
}
h2 {
    font-size: var(--ifm-h2-font-size);
}
```

It's not clear to me where these are actually defined. I looked in `docs/src/css` but nothing jumps out at me. It's possible there is some plugin that we are using that is causing this. 

### Dagger version

any

**Comment by marcosnils** [comment]
thing I found the issue Lev and seems like we are incorrectly using the markdown headings in our pages (https://docusaurus.io/docs/next/markdown-features/toc). 

Apparently, pages generally only contain a single `h1` heading which is the main title and then go down from there. I've observed that we're currently using two h1 heading in our pages which basically breaks the formatting and the standard layout. 
I guess what's left to do here is maybe close this and to fix our pages.
